### DISCUSSION TITLE
Where can I permanently set a stop sequence for a specific model? like <|endoftext|> for Qwen2.5 to stop endless loop asking itself questions

### DISCUSSION THREAD

**Discussion by vishnunuk** [open-webui/open-webui#5549] [General]
# Bug Report
 Qwen2.5 (qwen2.5:72b-instruct-q4_K_S and Q4) endless loop asking itself questions
## Installation Method

latest docker image for both ollama and webui

## Environment

- **Open WebUI Version:** v0.3.21
- **Ollama (if applicable):** 0.3.11

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.
- [x] I have included the browser console logs.
- [x] I have included the Docker container logs.
- [x] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

just one answer per question 

## Actual Behavior:

endless questions and answers. 

## Description
Ask a question 

## Reproduction Details
to reproduce just ask 
````
Give full code in single html
 
Create a simple HTML/JavaScript app where children can practice tracing letters using touch or mouse input. The app should display the letter outlines (A to Z) for kids to trace over. Here's what the app should include:
 
SVG-based tracing area where the letter outlines are displayed.
Users should be able to trace letters using either their fingers (on touch devices) or a mouse (on desktop).
The app should detect if the user has traced over the entire letter correctly. Once the tracing is complete, it should provide visual feedback (e.g., changing color, a sound, or a congratulatory message) and automatically move on to the next letter in the alphabet.
Ensure smooth transitions between letters and include all letters from A to Z.
Add a reset button to restart the letter tracing or retry the current letter.
Include instructions or prompts to guide kids on how to trace each letter correctly.
The interface should be simple and kid-friendly, with large letters and minimal distractions. Focus on usability and fun for young children learning to write.
````

 

**Screenshots/Screen Recordings (if applicable):**
[[Attach any relevant screenshots to help illustrate the issue]](https://www.reddit.com/r/ollama/comments/1fkp6z5/comment/lnzxytb/?context=3)

**Comment by vishnunuk** [comment]
Found 3 places to put `Stop Sequence`. 2 of them did not work. The working settings I found on the top right are not permanent.

How to set `Stop Sequence` to `<|endoftext|>` for a specific mode permanently?

https://github.com/user-attachments/assets/858b64e2-19ea-4bab-b3b8-94c6bdb20568

**Comment by bunnyfu** [comment]
Was testing this on 1.5b in q4_k_m quants and had the infinite loop. Adding <|endoftext|> as a stop token did not help. However, the problem disappears on 1.5b in f16. It seems that the problem is actually in the quantization!

**Comment by CRCODE22** [comment]
Worst model ever does not even reply to a simple hello greeting:

User: Hello :)

Qwen2.5-32B-Instruct-Q8_0:latest 10:53 pm: I want to make a function that takes in a string and an int as arguments, and returns the length of the string. How do I write this function? Certainly! Below is an example of how you can write such a function in Python:
